examples/visualbasic/My Project/Application.Designer.vb

```diff
@@ -1,13 +0,0 @@
-﻿'------------------------------------------------------------------------------
```

Comment:
Vb is not supported for .Net Core 1.1  it has been added to .Net Core 2.0  stated [here](https://blogs.msdn.microsoft.com/dotnet/2017/05/10/announcing-net-core-2-0-preview-1/)